winml/lib/Api.Image/ImageConversionHelpers.cpp

```diff
@@ -5,351 +5,347 @@
 #include "inc/ImageConversionHelpers.h"
 
 using namespace Microsoft::WRL;
-using namespace Windows::AI::MachineLearning::Internal;
 using namespace Windows::Graphics::DirectX::Direct3D11;
```

Comment:
why did you leave this one? can you remove it too?

Reply:
Windows::Graphics::DirectX::Direct3D11 is the native namespace, I only aliased the winrt:: ones.